.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "node": true
+  },
+  "extends": ["plugin:react/recommended", "google"],
```

Comment:
I added the google eslint config, and also with support for React.

I'm not sure if this is enough, but we can also add special rules if needed

Reply:
Awesome! I think this looks good for now, and you can always add more later as you see fit.

Reply:
Great, maybe we can discuss some React related rules in the future